package.json

```diff
@@ -12,10 +12,13 @@
   "keywords": [
     "trailpack",
     "trails",
-    "trailjs"
+    "trailjs",
+    "footprints",
+    "rest",
+    "restfull"
```

Comment:
Less than French but yeah sometime it's hard lol, sorry about this it's fixed now :)